File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        last = len(s)-1
        while 0 <= last and s[last] == " ":
            last -= 1
            
        count = 0    
        while 0 <= last and s[last] != " ":
            count += 1
            last -= 1
        
        return count

File: 0058-length-of-last-word/0058-length-of-last-word.go
func lengthOfLastWord(s string) int {
    last := len(s)-1
    for 0 <= last && s[last] == ' ' {
        last--
    }
    
    count := 0
    for 0 <= last && s[last] != ' ' {
        last--
        count++
    }
    
    return count
}